app/javascript/src/components/Dashboard/Contacts/ContactTable/ContactTable.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react";
```

Comment:
You are not paying attention to the filenames. 

Read:
- https://academy.bigbinary.com/learn-react/pages/structuring-code
- https://academy.bigbinary.com/learn-react/pages/naming-conventions-in-react